README.md

```diff
@@ -547,11 +588,11 @@ int main(int argc, char** argv) {
 
 The following example is a full implementation of a keyboard publisher
 that takes input from a Linux block device and publishes the low-level
-keyboard events to a ROS topic '/keyboard'
+keyboard events to a ROS topic '/keyboard'. For more information see [rxros_examples](https://github.com/rosin-project/rxros_examples)
```

Comment:
```suggestion
keyboard events to a ROS topic '/keyboard'. For more information see [rxros_examples](https://github.com/rosin-project/rxros_examples).
```

Reply:
And another comment: should we include these two examples here verbatim?

It might also be an option to address rosin-project/rxros_examples#7 and discuss these examples in that readme.

Reply:
I actually think the examples makes a nice completion to the README file. I think/hope they give the reader a more complete view of how some of the essential observables and operators are used together in a node.